app/controllers/users_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def create
+    user = User.create({ email: params[:email], name: params[:name], organization: params[:organization] })
+    render status: :created, json: UserPresenter.present(user)
+  end
+
+  def user_exists
+    user = User.find_by_email(params[:email])
+    render json: { user_exists: user ? true : false }
+  end
```

Comment:
We actually shouldn't expose an unprotected endpoint like this, since this is a security/privacy vulnerability. The security best practice is to not reveal anything about a user unless that user is authenticated. This even applies to things like registering an account and resetting passwords. See https://github.com/OWASP/CheatSheetSeries/blob/90ed5b2acc0516cf7f0e87a4029fba54b9020ade/cheatsheets/Authentication_Cheat_Sheet.md#authentication-and-error-messages

I haven't fully read through your frontend PR yet, but if possible, if this is only called in contexts where the user has already authenticated against Auth0, could we change this so that:

1. It's only accessible to authenticated users, and
2. It can only return results for the same email address that was authenticated by Auth0? I don't know what data is stored in the auth token that Auth0 will send back to the app, but we can only use the data within that token in order to look up whether a User record has been created in our app, since we can only trust that the data contained within the token is authenticated, not external parameters like the `email` param you have here.

    If Auth0 gives us an opaque identifier rather than an email address, then we'll want to store that ID in our database instead of (or maybe in addition to) the user's email.